src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_eventhub_commands_namespace_byok_test.py

```diff
@@ -31,7 +31,7 @@ def test_eh_namespace_byok(self, resource_group):
             'tags2': {'tag2=value2'},
             'sku': 'Standard',
             'tier': 'Standard',
-            'clusterarmid': '/subscriptions/326100e2-f69d-4268-8503-075374f62b6e/resourceGroups/prod-by3-533-rg/providers/Microsoft.EventHub/clusters/PMTestCluster',
+            'clusterarmid': '/subscriptions/6898adc8-5045-473d-a1bf-7012564f43cb/resourceGroups/prod-by3-533-rg/providers/Microsoft.EventHub/clusters/PMTestCluster',
```

Comment:
Why change this subscription?

Reply:
I recorded this live and it changed to the subscription I used.